src/Handler/TagHandler.php

```diff
@@ -73,7 +61,11 @@ public function getTagsHeaderName()
      */
     public function getTagsHeaderValue()
     {
-        return implode(',', array_unique($this->escapeTags($this->tags)));
+        if (empty($this->tags)) {
+            return '';
+        }
+
+        return $this->client->getTagsHeaderValue($this->tags);
```

Comment:
the logic of moving it to client was that client is only one knowing how header should look like to be able to work. But I have no problems standardizing header name and moving that back to ResponseTagger.